alpha/apps/kaltura/lib/batch2/bcdl/kBusinessConvertDL.php

```diff
@@ -22,6 +22,19 @@ public static function replaceEntry(entry $replacedEntry, entry $replacingEntry
 			return;
 		}
 
+		$lock = kLock::create("replacement_" . $replacedEntry->getId() . "_" . $replacingEntry->getId());
+		if ($lock && !$lock->lock())
+		{
+			KalturaLog::debug("Could not add a lock");
+			return;
+		}
+
+		if($replacingEntry->getSyncFlavorsOnceReady())
+		{
+			KalturaLog::debug("Function already ran from a different process");
```

Comment:
Use '  + add entry ID + flavor id to the message so it will be more useful in debug